test/e2e/utils/new.go

```diff
@@ -38,8 +39,15 @@ func NewDatadogAgentDeployment(ns, name, image string, options *NewDatadogAgentD
 			AppKey: "sgfggtdhfghfghfghfgbdfdgs",
 		},
 		Agent: &datadoghqv1alpha1.DatadogAgentDeploymentSpecAgentSpec{
-			Image:              datadoghqv1alpha1.ImageConfig{},
-			Config:             datadoghqv1alpha1.NodeAgentConfig{},
+			Image: datadoghqv1alpha1.ImageConfig{},
+			Config: datadoghqv1alpha1.NodeAgentConfig{
+				Resources: &v1.ResourceRequirements{
+					Requests: v1.ResourceList{
+						v1.ResourceCPU:    resource.MustParse("0"),
+						v1.ResourceMemory: resource.MustParse("0"),
+					},
```

Comment:
not sure if we want to keep these too

Reply:
it will help de schedule the daemonset on low test cluster